source.go

```diff
@@ -131,3 +149,25 @@ func (s *Source) Teardown(ctx context.Context) error {
 	}
 	return nil
 }
+
+func (s *Source) getTableColumns(conn *pgx.Conn) ([]string, error) {
+	query := fmt.Sprintf("SELECT column_name FROM information_schema.columns WHERE table_name = '%s'", s.config.Table[0])
+
+	rows, err := conn.Query(context.Background(), query)
```

Comment:
We have a nice context that we get from `Open`, please use that, in case the query for some reason takes longer than expected and Conduit wants to abort it.

Reply:
oh that was definitely unintentional, thanks for noticing it!